src/media/js/apps.js

```diff
@@ -122,6 +122,18 @@ define('apps',
         return product[COMPAT_REASONS];
     }
 
+    function transform(app) {
+        // Normalize content types.
+        if (app.doc_type == 'website') {
+            app.isWebsite = true;
+            app.name = app.short_title || app.title;
```

Comment:
The fields are going to be `short_name` and `name`, per https://bugzilla.mozilla.org/show_bug.cgi?id=1162325 - I just need to merge the PR. `title` will stay but will be useless to the frontend (it's the original title attribute for the website)